linera-shared-storage/tests/store_test.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// The store created by the create_shared_test_store
+/// are all pointing to the same storage.
+/// This is in contrast to other storage that are not
+/// persistent (e.g. memory) or uses a random table_name
+/// (e.g. rocksDb, DynamoDb, ScyllaDb)
+/// This requires two changes:
+/// * Only one test being run at a time with a semaphore.
+/// * After the test, the storage is cleaned
+
```

Comment:
```suggestion
```
…or should it be the module documentation instead, and use `//!`?

Reply:
I do not think, it should be in the module documentation, because it really belongs to the test section.